test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,54 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+{
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+// Test that setUnrefTimeout correctly passes arguments to the callback
+{
+  const maxArgsNum = 4;
+  for (let i = 0; i < maxArgsNum; i++) {
+    const results = [];
+    const inputArgs = [];
+    // set the input argument params
+    for (let j = 0; j <= i; j++) {
+      inputArgs.push(j);
+    }
+
+    const timer = setUnrefTimeout(common.mustCall((...args) => {
+      // check the number of arguments passed to this callback.
+      strictEqual(args.length, i + 1,
+                  `arguments.length should be ${i + 1}.` +
+                  `actual ${args.length}`
+      );
+      for (const arg of args) {
+        results.push(arg);
+      }
+    }), 1, ...inputArgs);
```

Comment:
`clearTimeout(testTimer);` here.

Reply:
Thank you! But if I fix this as you requested, it occures an error.
I fixed as below 
https://github.com/nodejs/node/pull/20945/commits/ce868dd6c3f64d389ae42f11b0c16a0d63d74cbb